src/protobuf-net-data/protobuf-net-data.csproj

```diff
@@ -6,8 +6,8 @@
     <DebugType Condition="$(codecov) != 'true'">embedded</DebugType>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <IncludeSymbols>false</IncludeSymbols>
-    <TargetFrameworks>net20;net30;net35;net40;net45;netstandard2.0</TargetFrameworks>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
This was necessary because of many warnings about obsolete methods. Some small typo but also some larger problems with static methods of ProtoWriter and ProtoReader like Create, StartSubItem and EndSubItem.

Reply:
Warnings were fixed and threat warnings as errors re-enabled.